delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -56,7 +56,7 @@ final class Files(
     storages: FetchStorage,
     storagesStatistics: StoragesStatistics,
     remoteDiskStorageClient: RemoteDiskStorageClient,
-    config: StorageTypeConfig
+    s3Client: S3StorageClient
```

Comment:
Didn't get round to it yet but I was planning to pull out the file operations like `SaveFile` - it will need some refactoring to the interface. The clients should really be abstracted away from here